src/app/pages/users/user/user.component.ts

```diff
@@ -303,6 +307,9 @@ export class UserComponent extends AbstractTabComponent implements OnInit {
     if (!this.currentUserID) {
       return;
     }
+    // TODO: Fix setSearch with userIDs as param
+    this.windowService.setSearch('userID', (this.currentUserID).toString())
```

Comment:
@LucasBrazi06 @ClaudeROSSI is this the right way to handle the current user ?

Reply:
`this.centralServerService.getLoggedUser().id` is the right way.

Reply:
see comment above